configs/spec-references.ts

```diff
@@ -0,0 +1,523 @@
+/**
+ * CENTRALIZED ETHSPECIFY TAGS
+ *
+ * This file maps Lodestar code components to Ethereum specification references.
+ * No markers or comments are needed in the actual implementation files.
+ *
+ * Each entry contains:
+ * 1. A component identifier (e.g., class name, function name)
+ * 2. The corresponding ethspecify tag
+ * 3. Optionally, a file path for reference (not used by ethspecify)
+ */
+
+export const SpecReferences = [
+  // Functions
+  /**
+   * @spec compute_fork_digest
+   */
+  {
+    component: "compute_fork_digest",
+    filePath: "packages/config/src/genesisConfig/index.ts",
+    specTag: `<spec fn="compute_fork_digest" hash="8b33f64d" />`,
+  },
+  /**
+   * @spec compute_fork_version
+   */
+  {
+    component: "compute_fork_version",
+    filePath: "packages/state-transition/src/util/domain.ts",
+    specTag: `<spec fn="compute_fork_version" hash="6d472038" />`,
+  },
+  /**
+   * @spec compute_proposer_index
+   */
+  {
+    component: "compute_proposer_index",
+    filePath: "packages/state-transition/src/util/seed.ts",
+    specTag: `<spec fn="compute_proposer_index" hash="74c8109d" />`,
+  },
+  /**
+   * @spec get_attesting_indices
+   */
+  {
+    component: "get_attesting_indices",
+    filePath: "packages/state-transition/src/cache/epochCache.ts",
+    specTag: `<spec fn="get_attesting_indices" hash="3304d67e" />`,
+  },
+  /**
+   * @spec get_beacon_committee
+   */
+  {
+    component: "get_beacon_committee",
+    filePath: "packages/state-transition/src/cache/epochCache.ts",
+    specTag: `<spec fn="get_beacon_committee" hash="f04db100" />`,
+  },
+  /**
+   * @spec MAX_BLOBS_PER_BLOCK
+   */
+  {
+    component: "MAX_BLOBS_PER_BLOCK",
+    filePath: "packages/config/src/forkConfig/index.ts",
```

Comment:
the file reference here is no correct, and this is a constant, not a function, the previous function was removed from spec and replaced by `get_blob_parameters`